### PR TITLE
Ensure image exists before bumping to it

### PR DESCRIPTION
**Pull request by fejta**
/assign @BenTheElder

### CODE REVIEW

File: prow/bump.sh [hunk 1]
```diff
@@ -34,6 +34,7 @@ new_version="v$(date -u '+%Y%m%d')-$(git describe --tags --always --dirty)"
 for i in "$@"; do
   echo "program: $i"
   echo "new version: $new_version"
+  gcloud docker -- pull "${PREFIX:-gcr.io/k8s-prow}/${i}:${new_version}"
```
Comment (cjwagner) [inline prow/bump.sh]: This script needs `set -e` in order for a non-zero return code to exit the script.

Comment (BenTheElder) [reply prow/bump.sh]: We should copy over the boilerplate bash set stuff to this script anyhow.

Comment (fejta) [reply prow/bump.sh]: Done

Comment (BenTheElder) [reply prow/bump.sh]: we should consider doing this without an actual pull someday (using the registry API?) unfortunately there's no standard way to do this :(

**Pull request merged**